operations/sftpgo-authentication-service/.env.example

```
@@ -0,0 +1,49 @@
+LOG_LEVEL="DEBUG" # DEBUG, INFO, WARN, ERROR
+PORT="9090" # Port to listen on
+HOOK_API_KEY="your-hook-api-key" # API key for authenticating hooks
+
+#External-Auth Hook Configuration (web attachment access path)
+#Validates the x-jwt-assertion the CSM backend forwards as the password on
+#SFTPGo's GET /api/v2/user/token. Optional: if unset, the service still starts
+#and the pre-login/keyboard-interactive hooks keep working, but the
+#/external-auth-hook route fails closed (503) until these are set.
+AUTH_JWKS_ENDPOINT="https://api.asgardeo.io/t/yourorg/oauth2/jwks"
+AUTH_ISSUER="https://api.asgardeo.io/t/yourorg/oauth2/token"
+AUTH_AUDIENCE="your-audience-1,your-audience-2" # comma-separated, OR-matched; leave unset to accept any audience
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AUTH_AUDIENCE key should go before the AUTH_ISSUER key

(UnorderedKey)